asynq/debug.py

```diff
@@ -78,8 +77,8 @@ def format_error(error, tb=None):
     result = ''
     if hasattr(error, '_traceback') or tb is not None:
         tb = tb or error._traceback
-        result += '\n\nTraceback:\n%s' % ''.join(format_tb(tb))
```

Comment:
It's a public interface, so I'd rather not remove it unless it's really harmful.

`format_tb`'s docstring is nonsense though so maybe I should fix that.